src/document.h

```diff
@@ -70,6 +70,9 @@ typedef struct GeanyFilePrefs
 GeanyFilePrefs;
 
 
+#define GEANY_TYPE_DOCUMENT (document_get_type())
+GType document_get_type (void);
```

Comment:
Am 22.06.2016 um 17:31 schrieb Colomban Wendling:
> In src/document.h
> <https://github.com/geany/geany/pull/1038#discussion_r68075614>:
>
>> @@ -70,6 +70,9 @@ typedef struct GeanyFilePrefs
>>  GeanyFilePrefs;
>>
>>
>> +#define GEANY_TYPE_DOCUMENT (document_get_type())
>> +GType document_get_type (void);
>
> @kugel- <https://github.com/kugel-> something to say on this?
>

They are used in the g_signal_new() calls in geanyobject.c. What's wrong 
with them?